README.md

```diff
@@ -118,6 +118,33 @@ dependencies {
 }
 ```
 
+### Specifying a Checker Framework version
+
+To change the version of the Checker Framework used by the plugin, you can add
```

Comment:
The README needs to say what version of the Checker Framework is used by default.